fix: add v1.4.1 known issue: upgrade stuck in "Waiting Reboot" state (#715)
harvester/docs: docs/upgrade/v1-4-0-to-v1-4-1.md

Pass 2: Checking directory structure

  [...a list of various different errors may appear here...]

e2fsck: aborted

COS_ACTIVE: ********** WARNING: Filesystem still has errors **********
```

Finally, check partition sizes:

```
# lsblk -o NAME,LABEL,SIZE
NAME   LABEL             SIZE
loop0  COS_ACTIVE          3G
sr0                     1024M
vda                      250G
├─vda1 COS_GRUB           64M
├─vda2 COS_OEM            64M
├─vda3 COS_RECOVERY        4G
├─vda4 COS_STATE           8G
└─vda5 COS_PERSISTENT  237.9G
vdb    HARV_LH_DEFAULT   128G
```

In the above output, the COS_STATE partition is 8G in size. In this specific case (a system upgraded to v1.4.1, which now has a corrupt `active.img`) this a good indicator that there initially wasn't enough free space on that parition for the upgrade to succeed.

To fix this problem:

1. If you have two or more nodes, on each of the remaining nodes that are yet to be upgraded, log in via `ssh` and check the disk usage of `active.img` and `passive.img`:
   ```
   # du -sh /run/initramfs/cos-state/cOS/*
   1.7G    /run/initramfs/cos-state/cOS/active.img
   3.1G    /run/initramfs/cos-state/cOS/passive.img
   ```

>    If `passive.img` is using 3.1G of disk space, become root and run the following commands:

Reviewer: ```suggestion
   If `passive.img` consumes 3.1G of disk space, run the following commands using the root account:
```

   ```
   # mount -o remount,rw /run/initramfs/cos-state
   # fallocate --dig-holes /run/initramfs/cos-state/cOS/passive.img 
   # mount -o remount,ro /run/initramfs/cos-state
   ```

>    This will turn `passive.img` into a sparse file, which should then only be using 1.7G of disk space, the same as `active.img`. Doing this ensures that there will be enough free space on those other nodes to not get stuck again as the upgrade proceeds.
> 
> 2. On the stuck node, log in via `ssh`, become root and run the following commands:

Reviewer: ```suggestion
   `passive.img` is converted to a sparse file, which should only consume 1.7G of disk space (the same as `active.img`). This ensures that the other nodes have enough free space, preventing the upgrade process from becoming stuck again.

1. Access the stuck node via SSH, and then run the following commands using the root account:
```

   ```
   # mount -o remount,rw /run/initramfs/cos-state
   # cp /run/initramfs/cos-state/cOS/passive.img \
        /run/initramfs/cos-state/cOS/active.img
   # tune2fs -L COS_ACTIVE /run/initramfs/cos-state/cOS/active.img
   # mount -o remount,ro /run/initramfs/cos-state
   ```

>    This will copy your existing (clean) `passive.img` over the corrupt `active.img`, and set the label correctly so that it can be used.
> 
> 3. Reboot the stuck node. When you get to the GRUB boot screen, it will still default to "Harvester v1.4.1 (fallback)". Select the first entry instead ("Harvester v1.4.1"). Despite the version shown on the GRUB boot screen, this will actually now boot the system into Harvester v1.4.0.
> 
> 4. Copy `rootfs.squashfs` from the Harvester v1.4.1 ISO image to somewhere convenient on the stuck node. You can either mount the ISO directly on the stuck node, or you can mount it on another system and copy the file over via `scp`.
> 
> 5. Log in to the stuck node via `ssh`, become root, and run the following commands (replace `rootfs.squashfs` on the fourth line with whatever path you copied that file to):

Reviewer: ```suggestion
    The existing (clean) `passive.img` is copied over the corrupted `active.img` and the label is set correctly.

1. Reboot the stuck node, and then select the first entry ("Harvester v1.4.1") on the GRUB boot screen.

    The GRUB boot screen initially displays "Harvester v1.4.1 (fallback)" by default. Despite the displayed version, the system boots into Harvester v1.4.0.

1. Copy `rootfs.squashfs` from the Harvester v1.4.1 ISO to a convenient location on the stuck node.

    The ISO can be mounted either on the stuck node or on another system. You can copy the file using the `scp` command.

1. Access the stuck node via SSH, and then run the following commands using the root account:
```
